lib/commands/config/index.js

```diff
@@ -5,6 +5,16 @@ const Command           = require('../../command');
 const advancedOptions   = require('./advanced');
 
 class ConfigCommand extends Command {
+    static configureSubcommands(commandName, commandArgs, extensions) {
+        return commandArgs.command({
+            command: 'get <key>',
+            handler: (argv) => this._run(`${commandName} get`, argv, extensions)
```

Comment:
What exactly is the reasoning behind renaming the command here? Is it just for convenience or does yargs  somehow use it?

Reply:
So the reason that’s passed in is the debug output uses that command name to say the error occurred running command “x”. Probably not a _huge_ deal with these small commands, but it’s probably worth it to be consistent 🙂

Reply:
Got it - I guess it just feels weird to me that the command name is `ghost config (s|g)et`